src/components/ColorRampDropdown.module.css

```diff
@@ -42,20 +55,32 @@
   flex-direction: column;
   background-color: var(--color-button);
 
-  --outline-width: 1px;
+  --border-width: 1px;
+  --outline-width-selected: 1px;
+  --outline-width-unselected: 0px;
   --radius: 6px;
-  padding: var(--outline-width) var(--outline-width);
-  gap: var(--outline-width);
+  padding: var(--border-width) var(--border-width);
+  gap: var(--border-width);
   border-radius: var(--radius);
 
-  --button-radius: calc(var(--radius) - var(--outline-width));
+  --button-radius: calc(var(--radius) - var(--border-width));
 
   transform-origin: top;
   visibility: hidden;
   transform: scaleY(0.9);
   opacity: 0;
 
-  transition: all ease-in 0.2s 0.25s;
+  transition: all ease-in 0.2s 0.25s, gap 0s;
+}
+
+/** Overrides for when the categorical dropdown is being used, instead of
+ * the normal color ramp version. Apply an extra border to each element,
+ * making it thicker when hovered.
+ */
```

Comment:
did you try adding this extra outline to the continuous ramp dropdown so that it wouldn't have to be special-cased?

Reply:
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/6703ff82-262a-484b-8d92-7bcac8a0cd1f)

I did but I wasn't a huge fan? It feels like it adds a lot of visual noise.